kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/k8s/KubeClient.java

```diff
@@ -203,6 +229,25 @@ public Deployment getDeployment(String namespaceName, String deploymentName) {
         return client.apps().deployments().inNamespace(namespaceName).withName(deploymentName).get();
     }
 
+    /**
+     * Gets deployment name by prefix.
```

Comment:
Why's this useful?   Deployments normally have well known names.     They are not like pod names that have trailing stuff.